api/src/main/java/org/openmrs/Person.java

```diff
@@ -65,7 +65,7 @@
 @Entity
 @Table(name = "person")
 @Inheritance(strategy = InheritanceType.JOINED)
-public class Person extends BaseChangeableOpenmrsData {
```

Comment:
Hi @dkayiwa  as per the ticket,class Person was extending BaseOpenmrsData but currently the class Person extends  BaseChangeableOpenmrsData.Could this ticket still be of value?